src/components/shared/buttons/ExportButton.vue

```diff
@@ -46,12 +61,19 @@
             }
         },
 
+        computed: {
+            exportUrl() {
+                const regex = /\/$/gm
```

Comment:
Just wondering why we are forcibly removing the trailing slash?
`//test.com/route/?foo=bar` is a valid route and isn't always a synonym for `//test.com/route?foo=bar` depending on the server config.

Reply:
I didn't realise that the top one was valid TBH, i have removed this logic. thanks